samples/batch/batch-ingestion-client/RealtimeTranscription/Helpers/ResultConversionHelper.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="ResultConversionHelper.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace RealtimeTranscription
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Xml;
+    using Connector;
+    using Microsoft.Extensions.Logging;
+
+    public static class ResultConversionHelper
+    {
+        public static SpeechTranscript CreateBatchResultFromRealtimeResults(string sourceName, List<JsonResult> realtimeResults, ILogger logger)
```

Comment:
although we are using the same engine, but the results returned by realtime transcrption and batch transcription might still be different in some cases. It could cause confusion to create result file using batch format but taking results from realtime one. For example, batch transcription uses different default mode for diarization, and also does not write results with nomatch/error into the result file. There might be other difference.

Reply:
For now, I have filtered empty results!